examples/todos/app.js

```diff
@@ -1,19 +1,38 @@
-
-/**
- * Module dependencies.
- */
-
-var express = require('express')
-  , path = require('path')
-  , morgan = require('morgan');
-
+var http = require('axios');
+var read = require('read-yaml');
+var PouchDB = require('pouchdb');
+var express = require('express');
+var path = require('path');
 var app = express();
+var config = read.sync('./config.yml');
 
+const DB_URL = `${config.protocol}${config.domain}:${config.port}${config.dbServerEndpoint}`
+const DB_ADMIN_URL = `${config.protocol}${config.admin.username}:${config.admin.password}@${config.domain}:${config.port}${config.dbServerEndpoint}`
+
+app.use(config.dbServerEndpoint, require('express-pouchdb')(PouchDB.defaults({prefix: './db/'})));
+app.listen(config.port);
 
-app.use(morgan('dev'));
 app.use(express.static(path.join(__dirname, 'public')));
-app.use('/db', require('../../'));
 
+async function setup() {
+  
+  // Set up the admin user.
```

Comment:
``app.couchConfig.set('admins', config.admin.username, config.admin.password, function (err, previousValue) {})``

might be more robust. Less chance, although that chance is already small admittedly, of someone hijacking your database before the admin is set. Note that I did not run the code, so using it would require testing it.

Reply:
>  Less chance, although that chance is already small admittedly, of someone hijacking your database before the admin is set.

Lol, ya, I've thought about that but not overly worried. I didn't know about `app.couchConfig.set`. Cool stuff!

Reply:
`app.couchConfig.set` does not seem to work. It makes it crash with "TypeError: refreshUsersDBImpl is not a function". What is weird is that it still set up the config file properly...

Reply:
It seems what's happening in this case is that the users data base isn't ready yet, but setting up an admin will still cause it to try to update it.

Reply:
I've created an issue here #341